plugins/omar-wms-plugin/grails-app/services/omar/wms/WebMappingService.groovy

```diff
@@ -268,12 +285,18 @@ class WebMappingService implements InitializingBean
 
   def getMap(GetMapRequest wmsParams)
   {
-    def otherParams = [startDate: new Date()]
+    def otherParams = [startTime: new Date(), internalTime: new Date(), procTime: new Date()]
 
     log.trace "getMap: Entered ................"
     otherParams.startTime = System.currentTimeMillis()
     otherParams.internalTime = otherParams.startTime
 
+    // log timestamp (can be added to otherParams if needed)
+    TimeZone.getTimeZone('UTC')
+    Date date= new Date()
+    String newdate=date.format("YYYY-MM-DD HH:mm:ss.Ms")
+    log.info "getMap timestamp " + newdate
```

Comment:
For clarity, I would roll this entire call up into one line and remove the two assignments above:
log.info("getMap timestamp " + new Date().format("YYYY-MM-DD HH:mm:ss.Ms"))

Reply:
Done